lib/ret/media_resolver.ex

```diff
@@ -16,7 +16,7 @@ defmodule Ret.MediaResolver do
 
   alias Ret.{CachedFile, MediaResolverQuery, Statix}
 
-  @ytdl_valid_status_codes [200, 302, 500]
+  @ytdl_valid_status_codes [200, 500]
```

Comment:
I was viewing htis as responses we expect to get back when things are working correctly. While its true that a 400 error would probably not benefit from retrying I also don't expect it to happen since our queries are static.